GiEnJul/ClientApp/src/components/institution/InstitutionLocations.tsx

```diff
@@ -36,12 +36,17 @@ const Location:FC<LocationProps> = ({
         setIsErr(viewError && !isValid);
     }, [value, setIsValidLocation, setIsErr, viewError])
 
-    return(
+    const [locationList, setLocationList] = useState<string[]>(['Ingen lokasjoner']);
```

Comment:
I think it would be nice to initialize this with an empty array, instead of "Ingen lokasjoner". This way the "Ingen lokasjoner" wouldn't be used as a location if no active events. I would suggest to change the options in the SelectValidator for cases with no locations, and use an empty string as value. This way the user can't proceed if "Ingen lokasjoner" is chosen.

Edit: I see that the "Ingen lokasjoner" will never be used, because it's overwritten by the useEffect below. The handling of empty arrays should be done though.

Reply:
Giver: Currently a spinner is shown for the dropdown as the list is waiting to be populated.

Institution: A text displaying that no locations are currently active is shown, and that the user should try again later.